PennyMe.xcodeproj/project.pbxproj

```diff
@@ -488,13 +492,13 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = "AppIcon-1";
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = H8Z8J2BM93;
+				DEVELOPMENT_TEAM = AJ8J4VXPCN;
 				INFOPLIST_FILE = PennyMe/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = PennyMe;
+				PRODUCT_BUNDLE_IDENTIFIER = PennyMe0.1;
```

Comment:
Did you intentionally change the identifier here? Not sure on the implications of this but I think this should stay as it was.

Reply:
I had to change it in order to make it run on my phone.. I can test again with the previous identifier